### PR TITLE
demo mxnet

### PR DESCRIPTION
**Pull request by daming-lu**

### CODE REVIEW

File: demo/mxnet/mxnet_demo.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import mxnet as mx
+import numpy
+
+# 引入VisualDL
+from visualdl import LogWriter
+
+# 下载MNIST数据集
```
Comment (Superjomn) [inline demo/mxnet/mxnet_demo.py]: to english

Comment (daming-lu) [reply demo/mxnet/mxnet_demo.py]: done

File: demo/mxnet/mxnet_demo.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+test_iter = mx.io.NDArrayIter(mnist['test_data'], None, batch_size)
+prob = lenet_model.predict(test_iter)
+test_iter = mx.io.NDArrayIter(mnist['test_data'], mnist['test_label'], batch_size)
+
+# predict accuracy for lenet
+acc = mx.metric.Accuracy()
+lenet_model.score(test_iter, acc)
+print(acc)
+
```
Comment (Superjomn) [inline demo/mxnet/mxnet_demo.py]: use pylint to style python code.

File: demo/mxnet/TUTORIAL_CN.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+```
+python setup.py bdist_wheel
+pip install --upgrade dist/visualdl-0.0.1-py2-none-any.whl
+```
+
+## 开始编写训练MNIST的程序
+
+我们为您提供了一个演示程序 [mxnet_demo.py](./mxnet_demo.py)。里面展示了如何下载MNIST数据集以及编写MXNet程序来进行CNN的训练。MXNet的部分借鉴了MXNet[官方入门文件](https://mxnet.incubator.apache.org/tutorials/python/mnist.html)
+为了嵌入VisualDL程序，以便在MXNet训练时进行检测，我们需要声明一个logger实例：
```
Comment (jacquesqiao) [inline demo/mxnet/TUTORIAL_CN.md]: logger => log writer

File: demo/mxnet/TUTORIAL_CN.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+```
+logger = LogWriter(logdir, sync_cycle=10)
+```
+
+logger实例里面包含VisualDL的四个功能模块 Scalar， Image， Graph 以及 Histogram。这里我们使用 Scalar 模块：
```
Comment (jacquesqiao) [inline demo/mxnet/TUTORIAL_CN.md]: 没有graph模块

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**